lib/payment_calculator/ecf/contract/uplift_payment_calculations.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require "has_di_parameters"
+
+module PaymentCalculator
+  module Ecf
+    module Contract
+      module UpliftPaymentCalculations
+        extend ActiveSupport::Concern
```

Comment:
Why do we want it to be a concern? At the moment it is used in a single place, so we could just inline it?

Reply:
It's a module as are most parts of the calculator. Keeping them as separated mix-ins keeps the similar functionality grouped together. Extending it as a concern gives us access to the `included do` rather than having to write `def self.included(base); base.class_eval do....` etc.